src/celeritas/ext/Convert.geant.hh

```diff
@@ -3,54 +3,68 @@
 // See the top-level COPYRIGHT file for details.
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
-//! \file accel/detail/Convert.hh
+//! \file celeritas/ext/Convert.geant.hh
 //---------------------------------------------------------------------------//
 #pragma once
 
 #include <CLHEP/Units/SystemOfUnits.h>
 #include <G4ThreeVector.hh>
 
 #include "corecel/Types.hh"
+#include "corecel/cont/Array.hh"
 #include "orange/Types.hh"
 #include "celeritas/Quantities.hh"
 
 namespace celeritas
 {
-namespace detail
-{
 //---------------------------------------------------------------------------//
+/*!
+ * Convert a value from Geant4/CLHEP to Celeritas native units.
+ */
 template<class T>
 inline T convert_from_geant(T const& val, T units)
 {
     return val / units;
 }
 
 //---------------------------------------------------------------------------//
+/*!
+ * Convert a 3-vector from Geant4/CLHEP to Celeritas native units.
```

Comment:
Given that 'units' is a parameter technically this could be used to move to/from 'any' units system rather than just 'Geant4->Celeritas'